### ISSUE TITLE
edit messages

### ISSUE THREAD

**Issue by KiaraGrouwstra** [isamert/scli#219]
signal had recently added the possibility to [edit messages](https://www.reddit.com/r/signal/comments/14tcorc/beta_signal_can_now_edit_sent_messages/).
it would be nice to have such an option in scli as well.

**Comment by exquo** [comment]
Certainly, would be a good feature to have! It requires an upstream signal-cli implementation (DBus interface for sending edited messages): https://github.com/AsamK/signal-cli/issues/1420
